python/arcticdb/version_store/_store.py

```diff
@@ -2239,6 +2251,8 @@ def compact_incomplete(
               ``compact_incomplete``.
 
             To manually delete staged data, use the ``remove_incomplete`` function.
+        _stage_results : `Optional[List[StageResult]]`, default=None
```

Comment:
I would leave this undocumented, on the `library.py` too

Reply:
Alex had a comment about documenting this, but I agree it is better hidden for now